Misc/NEWS.d/next/Library/2019-12-04-15-56-28.bpo-38634.pq0ZWa.rst

```diff
@@ -0,0 +1,5 @@
+On Linux systems, when libpython is linked into a native application that
+uses it to start a Python interpreter with readline activated, if that
+native application was also linked to libedit, the readline native module
+used to crash due to conflicting implementation. The readline native module
+now detects this situation and adapt its behavior accordingly.
```

Comment:
Hum, even if the case occurs under very specific conditions, I suggest to simplify the NEWS entry. Something like:

```
The :mod:`readline` module now detects if Python is linked to libedit at runtime on all platforms.
Previously, the check was only done on macOS.
```